package.json

```diff
@@ -72,16 +72,17 @@
         "chai": "*",
         "chai-as-promised": "^7.1.1",
         "chai-http": "^4.3.0",
-        "electron": "^9.3.0",
-        "electron-builder": "^22.8.0",
+        "electron": "^20.3.3",
+        "electron-builder": "^23.6.0",
+        "electron-reloader": "^1.2.3",
         "husky": "^4.3.0",
         "mocha": "*",
         "prettier": "^2.1.1",
         "pretty-quick": "^3.0.0",
-        "sass": "^1.26.10",
-        "spectron": "^11.1.0"
```

Comment:
We'll probably need to find an alternative for `spectron` as it's deprecated since February. Maybe [playwright](https://github.com/microsoft/playwright)?

Reply:
OMG didn't know it has an e2e testing. Such a cool repo. Yeah I think playwright is [officially recommended](https://www.electronjs.org/blog/spectron-deprecation-notice#alternatives-to-spectron).

Reply:
Moved some of the tests to playwright in https://github.com/PicnicSupermarket/nepsnowplow/pull/30/commits/0ce1381dc6912470ccfb5e83f2e6e6dbb6dc83e0.